### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by masayag**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,23 +23,24 @@ Use the developer CLI commands by running:
...
   add         Add a new flotta resource
   completion  Generate the autocompletion script for the specified shell
-  delete      Delete a flotta resource
+  delete      Delete the flotta resource
   help        Help about any command
-  list        list flotta resources
-  start       Start a flotta resource
-  stop        Stop a flotta resource
+  list        List flotta resources
+  start       Start flotta resource
```
Comment (jordigilh) [inline README.md:39]: Starts a flotta resource, it's singular.

File: README.md [hunk 2]
```diff
@@ -20,23 +23,24 @@ Use the developer CLI commands by running:
...
+  stop        Stop flotta resource
```
Comment (jordigilh) [inline README.md:40]: same here: `Stop a flotta resource`

File: README.md [hunk 3]
```diff
@@ -20,23 +23,24 @@ Use the developer CLI commands by running:
...
 
 ```
+Usage:
+  flotta [command]
+
 Available Commands:
   add         Add a new flotta resource
   completion  Generate the autocompletion script for the specified shell
-  delete      Delete a flotta resource
+  delete      Delete the flotta resource
```
Comment (jordigilh) [inline README.md:36]: `the` or `a`?

File: README.md (hunk 1 above)
Comment (masayag) [reply README.md:39]: to justify my decision: did what kubectl did... :-) 
```
→ kubectl 
kubectl controls the Kubernetes cluster manager.

 Find more information at: https://kubernetes.io/docs/reference/kubectl/overview/

Basic Commands (Beginner):
  create        Create a resource from a file or from stdin
  expose        Take a replication controller, service, deployment or pod and expose it as a new Kubernetes service
  run           Run a particular image on the cluster
  set           Set specific features on objects
```

**Review by jordigilh — APPROVED** [review]
LGTM

**Pull request merged**